src/components/layouts/layout.tsx

```diff
@@ -361,57 +351,32 @@ export default function Layout(props: PropsWithChildren) {
 
                 <li className="-mx-6 ">
                   <Menu as="div" className="relative">
-                    <Menu.Button className="flex w-full items-center gap-x-4 p-1.5 px-6 py-3 text-sm font-semibold leading-6 text-gray-900 hover:bg-gray-50">
-                      <span className="sr-only">Open user menu</span>
-                      <Avatar className="h-8 w-8">
-                        <AvatarImage
-                          src={session.data?.user?.image ?? undefined}
-                        />
-                        <AvatarFallback>
-                          {session.data?.user?.name
-                            ? session.data.user.name
+                    <Link href='./userSettings'>
```

Comment:
Right now, i get a 404 error when i hit the name of the user on the bottom left. Can you fix that?

Reply:
It's my fault. Now, it works perfectly